definitions/npm/@testing-library/react_v10.x.x/flow_v0.104.x-/react_v10.x.x.js

```diff
@@ -245,17 +245,17 @@ declare module '@testing-library/react' {
     container?: HTMLElement,
     baseElement?: HTMLElement,
     hydrate?: boolean,
-    wrapper?: React.ComponentType,
+    wrapper?: React$ComponentType<any>,
   |};
 
   declare export function render(
-    ui: React.ReactElement<any>,
+    ui: React$Node,
```

Comment:
That's definitely not the same thing, a Node is really permissive compared to an Element.

Reply:
Looks like a poor copy/pasta from TS. 
It should be `ui: React$Element<any>`